nixos/doc/manual/configuration/modularity.xml

```diff
@@ -66,7 +66,7 @@ instance, if two modules define
 <command>nixos-rebuild</command> will give an error:
 
 <screen>
-The unique option `services.httpd.adminAddr' is defined multiple times, in `/etc/nixos/httpd.nix' and `/etc/nixos/configuration.nix'.
+The unique option 'services.httpd.adminAddr' is defined multiple times, in '/etc/nixos/httpd.nix' and '/etc/nixos/configuration.nix'.
```

Comment:
This is incorrect, because the original quotes are the actual output of that command. So please don't change the quoting style.